test_dataset.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import pandas as pd
+
+
+ALLOWED_COLUMN_NAMES = [
+    # record_id
+    "record_id",
+
+    # default metadata
+    "title",
+    "abstract",
+    "keywords",
+    "authors",
+    "year",
+    "date",
+    "doi",
+
+    # custom variables
+    "label_included",
+    "label_abstract_screening",
+    "duplicate_record_id",
```

Comment:
@J535D165 what do you mean with duplicate_record_id? is this the result of a script testing for duplicates?

Also, is `debug_label` allowed? see our documentation

Reply:
> @J535D165 what do you mean with duplicate_record_id? is this the result of a script testing for duplicates?

Yes @terrymyc adds this column to indicate duplicates without removing them from the dataset. The value is the record_id of the first duplicated entry in the dataset. 

> Also, is debug_label allowed? see our documentation

No this is not allowed. Datasets repo should be more or less independent of ASReview. Besides that, this column is on the list for deprecation in ASReview (almost there).